paddle/fluid/operators/conv_op.cc

```diff
@@ -167,17 +192,24 @@ void Conv2DOpMaker::Make() {
       .AsDispensable();
   AddOutput("Output",
             "(Tensor) The output tensor of convolution operator. "
-            "The format of output tensor is also NCHW.");
+            "The format of output tensor is also NCHW or NHWC.");
```

Comment:
It has same data layout and data type with the input.

Reply:
done